system/jlib/jlog.hpp

```diff
@@ -1342,4 +1348,301 @@ interface IComponentLogFileCreator : extends IInterface
 extern jlib_decl IComponentLogFileCreator * createComponentLogFileCreator(IPropertyTree * _properties, const char *_component);
 extern jlib_decl IComponentLogFileCreator * createComponentLogFileCreator(const char *_logDir, const char *_component);
 extern jlib_decl IComponentLogFileCreator * createComponentLogFileCreator(const char *_component);
+
+struct LogAccessTimeRange
+{
+    //%Y-%m-dT%H:%M:%S.%z%z
+    std::time_t startt = -1;
+    std::time_t endt = -1;
```

Comment:
wondering whether we use jlib CDateTime's here as elsewhere.
std::time_t may be ok, but we may need wrappers or helpers to validate (which could be part of CDateTime)

Reply:
done